.vscode/settings.json

```diff
@@ -0,0 +1,23 @@
+[
```

Comment:
I might be unfamiliar with vscode, but this file doesn't look like
isn't vscode settings file supposed to be a object, rather than an array with objects?

and while you are at it, you can also merge in my local sp7's settings.json with this one:
```json
{
  "typescript.tsdk": "./specifyweb/frontend/js_src/node_modules/typescript/lib",
  "terminal.integrated.cwd": "specifyweb/frontend/js_src/lib/",
  "prettier.prettierPath": "./specifyweb/frontend/js_src/node_modules/prettier",
  "todo-tree.tree.scanMode": "workspace only",
  "eslint.experimental.useFlatConfig": true,
  "eslint.workingDirectories": ["./specifyweb/frontend/js_src"],
  "search-node-modules.path": "/specifyweb/frontend/js_src/node_modules",
  "cSpell.words": ["Syncer"]
}
```